services/subnet_manager/src/test/java/com/futurewei/alcor/subnet/SubnetControllerTests.java

```diff
@@ -106,6 +106,8 @@ public void createSubnetState_create_pass () throws Exception {
                 .thenReturn(macResponse);
         Mockito.when(subnetService.allocateIpAddressForGatewayPort(anyString(), anyString(), anyString()))
                 .thenReturn(ipAddrRequest);
+        Mockito.when(subnetService.cidrToFirstIpAndLastIp(UnitTestConfig.cidr))
+                .thenReturn(new String[2]);
```

Comment:
does this actually test the first desired ip and the last desired ip?

Reply:
Nope, this is for the create pass test, we could add a new UT to test the first desired ip and the last desired ip